File: apps/Api.CardanoSounds/models/metadata.py
from dataclasses import dataclass
from typing import List
from models.soundprobability import SoundProbability

@dataclass
class Metadata:
    tx_hash: str
    token_name: str
    probabilty: float
    rarity: int
    sounds: List[SoundProbability]
    arweave_id_sound: str = ""
    ipfs_id_sound: str = ""
    arweave_website_uri: str = ""
    

File: apps/Csharp.CardanoSounds/CS.Models/metadata.ts
export interface Metadata {
	id: string;
	token_name: string;
	player: string;
	image: string;
	probability: number;
	rarity: string;
	sounds: Sound[];
	arweave_id_sound: string;
	ipfs_id_sound: string;
	arweave_website_uri: string;
}

export interface Sound {
	probability: number;
	filename: string;
	category: string;
}

export interface Metadata {

}